classes/DepositDAO.inc.php

```diff
@@ -262,15 +271,18 @@ public function getNeedStagingStatusUpdate($journalId) {
 			'SELECT *
 			FROM pln_deposits AS d
 			WHERE d.journal_id = ?
-			AND d.status & ? <> 0
-			AND d.status & ? = 0
-			AND d.status & ? = 0
+			AND (
+				d.status IS NULL
+				OR (
+					d.status & ? <> 0
+					AND d.status & ? = 0
+				)
```

Comment:
To pass through a status check, a deposit needs to be transferred and not preserved.
And an exception was added for deposits without a status (the field was already allowing nulls), but the real motivation was my decision to reset the status.